modules/training/README.md

```diff
@@ -1,7 +1,12 @@
 # Training Module
 
-The Training Module provides a platform to teach and certify the administration of instruments.
+## Purpose
+The training Module provides a platform to teach examiners how to administer instruments, as well as testing and certifying their ability to do so.
 
+## Intended Users
+Only users who are listed as examiners will be able to complete the training. Users who are not examiners but still have permission to view the Training Module will be able to see the training content, but will not be able to complete the quiz and become certified. You can edit the examiners through the Examiner Module.
```

Comment:
Can you:

1. Add a blank line between the headers and bodies
2. Fix the line lengths?

Reply:
Only the first sentence describes intended users, the others describes permissions and should be moved to the permissions section.